### PR TITLE
Fix typo in morphology doc

### PR DESCRIPTION
**Pull request by rickymwalsh**
## Description

Fixed the morphology doc which erroneously referred to dilation when describing the second step.

## Checklist

- [x] A descriptive but concise pull request title
- [ ] N/A - [Docstrings for all functions](https://github.com/numpy/numpydoc/blob/main/doc/example.py)
- [ ] N/A - [Unit tests](https://scikit-image.org/docs/dev/development/contribute.html#testing)
- [ ] N/A - A gallery example in `./doc/examples` for new features
- [x] [Contribution guide](https://scikit-image.org/docs/dev/development/contribute.html) is followed

### CODE REVIEW

**Review by mkcor — APPROVED** [review]
Good catch! Thanks, @rickymwalsh.

**Comment by mkcor** [comment]
Speaking of which, I admit that, as much as I understand "Also notice how the crack we added is mostly removed" (in context) and I can see it on the figure, I don't understand nor see "Notice how the white ellipses at the bottom get connected" (I can't see any difference in the "white ellipses at the bottom" between the two figures)...

**Review by jarrodmillman — APPROVED** [review]
CI failure is unrelated. Will rebase after #7612 is merged. Will merge this after that.

**Pull request merged**
